cmd/ocsp-updater/main.go

```diff
@@ -167,7 +184,14 @@ func getQuestionsForShardList(count int) string {
 	return strings.TrimRight(strings.Repeat("?,", count), ",")
 }
 
-func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Time, batchSize int) ([]core.CertificateStatus, error) {
+// findStaleOCSPResponses sends a goroutine to fetch rows of stale OCSP
+// responses from the database. Returns results on a channel and pipelined
+// to the next step.
+func (updater *OCSPUpdater) findStaleOCSPResponses(ctx context.Context, oldestLastUpdatedTime time.Time, batchSize int) <-chan core.CertificateStatus {
+	// staleQueue channel contains all stale ocsp responses that need
+	// updating.
+	staleQueue := make(chan core.CertificateStatus)
```

Comment:
I would highly recommend all of these 0-size channels be renamed to indicate they are _not_ queues, they are direct-handoff channels. Or, do the adjustments to let them be queues.